src/ray/raylet/reconstruction_policy.h

```diff
@@ -17,6 +17,8 @@ namespace ray {
 
 namespace raylet {
 
+using rpc::TaskReconstructionData;
```

Comment:
better do this in .cc file to avoid potential conflicts?

Reply:
Previously, those data structures in `gcs.fbs` were defined in global namspace. 
If I tried putting this in the `.cc` files, but ended up with adding `::rpc` everywhere in all the header files. So I reverted it back. 
It's not ideal, but not worse than before..